content/en/docs/ambient/architecture/index.md

```diff
@@ -48,18 +98,18 @@ By default, both will be accepted by ztunnel.
 Because plaintext requests will have no peer identity when Authorization Policies are evaluated,
```

Comment:
This feels like an abrupt statement and its odd to lead with an edge case rather than the more common stuff which you cover below.

We should contextualize 'plaintext' as 'traffic from sources not yet in the mesh' to be consistent with our terminology earlier.

Reply:
Updated the `plaintext` wording. Not exactly sure how to fix the abrupt statement seems okay for me....